CONTRIBUTING.md

```diff
@@ -25,7 +25,7 @@ yarn test
 ## Submitting bug reports
 
 * Search through issues to see if a previous issue has already been reported and/or fixed.
-* Provide a _small_ reproduction using a [plunker template](http://plnkr.co/edit/tpl:757r6L?p=preview) or github repo.
+* Provide a _small_ reproduction using a [plunker template](https://plnkr.co/edit/tpl:757r6L?p=preview) or github repo.
```

Comment:
You can replace the plunker template with our new StackBlitz template, https://stackblitz.com/edit/ngrx-seed

Reply:
The provided URL points to personal StackBlitz account. Do we have one that points to an ngrx account?

Reply:
This is the URL we use in the new issue templates.

Reply:
Replaced